docs/docs-ref-autogen/excel/excel/excel.workbookrangeareas.yml

```diff
@@ -41,8 +41,8 @@ properties:
     package: excel!
     fullName: areas
     summary: >-
-      Returns the `RangeAreasCollection` object. Each `RangeAreas` object in the
-      collection represents one or more rectangle ranges in one worksheet.
+      Returns the `RangeAreasCollection` object. Each `RangeAreas` in the
```

Comment:
Can they revert this change upstream? I think Ricky and I agreed that the original read better than the new version.

Reply:
Yeah, I've got that in my change list. I'll fix it in DT next time.